libethcore/KeyManager.cpp

```diff
@@ -95,10 +96,21 @@ bool KeyManager::load(string const& _pass)
 			{
 				h128 uuid(i[1]);
 				Address addr(i[0]);
-				if (m_store.contains(uuid))
+				if (uuid)
 				{
-					m_addrLookup[addr] = uuid;
-					m_keyInfo[uuid] = KeyInfo(h256(i[2]), string(i[3]));
+					if (m_store.contains(uuid))
+					{
+						m_addrLookup[addr] = uuid;
+						m_uuidLookup[uuid] = addr;
+						m_keyInfo[addr] = KeyInfo(h256(i[2]), string(i[3]), i.itemCount() > 4 ? string(i[4]) : "");
+					}
+					else
+						cwarn << "Missing key:" << uuid << addr;
+				}
+				else
+				{
+					// TODO: brain wallet.
```

Comment:
Since you implemented the brain wallet in this PR, is there something important to do here?